scripts/release.sh

```diff
@@ -67,6 +85,9 @@ if ( echo "$version" | grep -q "SNAPSHOT" ); then
     exit 1
 fi
 
+# Clean up the state at the beginning in case the previous run did not finish successfully
+clean_up_gh_pages
+
 if [ -z "${DRYRUN:-}" ]; then
     gradle_build_args="--no-build-cache --warning-mode all --refresh-dependencies clean build publishToMavenLocal"
```

Comment:
I've left a comment how to simplify this command on the PR that is already merged, we can update it here: https://github.com/apple/servicetalk/pull/2193#discussion_r854283209

Reply:
used `assemble` as recommended.